multi-ai-cli/core/cache.py

```
@@ -1,58 +1,23 @@
-
-    return target_real
-
+    return str(CACHE_DIR / f"{session_id}.json")
 
 def cache_load(session_id: str) -> Optional[List[Dict]]:
-    path = cache_path(session_id)
-    if os.path.exists(path):
-        with open(path, "r", encoding="utf-8") as f:
+    p = cache_path(session_id)
+    if os.path.exists(p):
+        with open(p) as f:
```

Pylint (4.0.7)

[warning] Using open without explicitly specifying an encoding

(W1514)